dataland-frontend/src/components/forms/parts/fields/PercentageFormField.vue

```diff
@@ -6,7 +6,7 @@
       :name="name"
       :validation-label="validationLabel ?? label"
       :validation="`number|between:0,100|${validation}`"
-      :placeholder="placeholder ?? 'Value in %'"
```

Comment:
I think the percentage form field should display a "%" after the value.
Current look: 
![image](https://github.com/d-fine/Dataland/assets/108792843/655c8820-d0ca-4bf0-bbe4-a17201d980b9)
Desired look:
![image](https://github.com/d-fine/Dataland/assets/108792843/df04c1ea-8c02-4407-929f-f43eaac99385)

Reply:
Will double check, might need a different story for it.

Reply:
Story created.